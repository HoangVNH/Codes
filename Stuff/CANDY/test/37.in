10
47
DDDXXXXDDDDXDVVXDXXXXVVXXDXXXXDDDDDDXXXXXXXXXXX
37
DDDXVVVVDDDDDDXXXXVVVDDDDDDXXXXXXXVVV
23
VVDDXXXXXXXXXXXXXXXXXXX
27
XXDDDDDXXXXXXDDDDDDVVVVDDVV
37
DDDDDDDDXXXXDDDDDDDXXXDDDVVDDDVVVVVVV
49
DDDXXXVDDDDDDVVVVVVVDDDXDDDVVVVVDDDDDDDDDDVVVDDVV
23
VVVVXXXXDDDXXXDVVXDDDDD
39
DDDDXXXVVVVVVXXXXXVVVVVXXXXXVVVDDDDDDDD
36
DXVVVXXXXVVVXXXVDDVVVVVXXXXDDXXXXVDD
20
VVVXXXDVVVVVXXDDDDDD
