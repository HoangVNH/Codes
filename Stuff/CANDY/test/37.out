46
36
22
26
35
48
22
38
35
19
